scripts/npm/remove-owner.sh

```diff
@@ -34,6 +34,4 @@ for f in packages/*; do
   fi
 done
 
-sed -i '' "/$username/d" NPM_OWNERS
-
-echo "Success."
+echo "Removed '${username}' from all packages. Remember to remove them from meta.json.";
```

Comment:
I was going to replicate the `sed` logic here but I think for the time it would actually save, this is a more pragmatic solution. 

`update-owners` doesn't add the user to `meta.json`, so I figured it's ok for this script to not actually remove users from `meta.json` either.